website/docs/d/schema_site_anp_epg_bulk_staticport.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "mso"
+page_title: "MSO: mso_schema_site_anp_epg_bulk_staticport"
+sidebar_current: "docs-mso-data-source-schema_site_anp_epg_bulk_staticport"
+description: |-
+  Data source for MSO Schema Site ANP EPG Bulk Static Port.
+---
+
+# mso_schema_site_anp_epg_bulk_staticport #
+
+Data source for MSO Schema Site ANP EPG Bulk Static Port.
+
+## Example Usage ##
+
+```hcl
+
+data "mso_schema_site_anp_epg_bulk_staticport" "static_port" {
+  schema_id     = data.mso_schema.schema1.id
+  site_id       = data.mso_site.site1.id
+  template_name = "Template1"
+  anp_name      = "ANP"
+  epg_name      = "DB"
+}
+
+```
+
+## Argument Reference ##
+
+* `schema_id` - (Required) SchemaID under which you want to deploy Static Port.
+* `site_id` - (Required) SiteID under which you want to deploy Static Port.
+* `template_name` - (Required) Template name under which Static Port is deployed.
+* `anp_name` - (Required) ANP name under which you want to deploy Static Port.
+* `epg_name` - (Required) EPG name under which you want to deploy Static Port.
+
+
+## Attribute Reference ##
+
+* `static_ports` - (Optional) A block representing a Static Port object. Type: Block.
+    * `path_type` - (Optional) The type of the static port.
```

Comment:
Should this be path type of the static port, or is type sufficient?

Reply:
if so same applies for resource doc

Reply:
resolved